docs/standards/hcs-1.md

```diff
@@ -26,57 +26,51 @@ description: The HCS-1 standard provides a systematic approach to encode, chunk,
-In summary, a valid HCS-1 Topic includes a SHA-256 hash of the file being uploaded in its memo, a submit key, and no admin key.
+1. **Include a SHA-256 Hash**: The memo should contain the file's SHA-256 hash, compression algorithm, and encoding. This ensures data validity by comparing the file hash to the memo.
+2. **Submit Key**: Topics must have a Submit Key. Topics without this key are invalid and ignored.
+3. **No Admin Key**: Topics with an Admin Key are invalid and ignored, ensuring data cannot be deleted, reducing protocol risk.
 
 ### Memo Structure
+The memo format is:
 
- The format for the memo is as follows:
-
- ```[hash]:[algo]:[encoding]```
-
- - [hash] is the SHA-256 hash of the file being uploaded before any compression
- - [algo] is the compression algorithm being used, for example, `zstd`
- - [encoding] is the encoding used to store the compressed file
-
- For example a valid memo would like:
-
- ```532eaabd9574880dbf76b9b8cc00832c20a6ec113d682299550d7a6e0f345e25:zstd:base64```
+```
+[hash]:[algo]:[encoding]
+```
 
- At this time, only zstd and base64 encoding are supported. If you'd like a different compression algorithm or encoding to be approved, please raise a pull request.
+- **[hash]**: SHA-256 hash of the file before compression.
+- **[algo]**: Compression algorithm (e.g., zstd).
+- **[encoding]**: Encoding method (e.g., base64).
 
-## Encoding and Chunking
+Example:
 
-Before uploading to HCS, file data must be formatted correctly by following these three steps
+```
+532eaabd9574880dbf76b9b8cc00832c20a6ec113d682299550d7a6e0f345e25:zstd:base64
+```
 
-1. The entire file should be compressed using zstd, this ensures cost-savings on all ends of the protocol in a lossless format.
-  - Plenty of implementations for zSTD exist, for example, using NodeJS you could use the MongoDB library: https://github.com/mongodb-js/zstd/tree/main
-  - ``` const compressedFile = await compress(file, 10);  ```
-  - We recommend using a compression level of 10, but any compression level should work.
-2. After compressing, the resulting data (typically a buffer) should be converted into a base64 string
-3. The final base64 string should chunked into segments no greater than 1024 bytes. Each chunk is encapsulated in a JSON object with two attributes:
+Only `zstd` and `base64` are currently supported.
 
-- `o`: The order index indicating the chunk's sequence in the overall file.
-- `c`: The chunk's content, a substring of the base64-encoded file. The first chunk aka, `o = 0` should include a data prefix for the mime type.
+### Encoding and Chunking
+Steps to format file data for HCS:
 
-The format for the data prefix is
-```data:[mimeType];base64```
-For example:
-```data:image/png;base64,```
+1. **Compression**: Compress the file using `zstd` (e.g., using MongoDB library in NodeJS).
```

Comment:
I think its worth mentioning that 10 is the recommended compression level, but other levels do work.